modules/ROOT/pages/troubleshoot-guide.adoc

```diff
@@ -96,6 +96,32 @@ as well as individual components.
+time="2020-02-25T17:42:15Z" level=info msg="  2. /opt/mule/conf/tokenpolicyapigw-4-2-2.yml (https://configuration-resolver.qax.msap.io/api/v1/deployments/add0b4d9-54f8-11ea-8202-85e61abacb9f/configurations/eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJNdWxlU29mdCAtIEFNQyAtIERlcGxveWVyIiwiZGVwbG95bWVudE5hbWUiOiJ0b2tlbnBvbGljeWFwaWd3LTQtMi0yIiwic291cmNlIjoiZGVwbG95ZXIiLCJ0eXBlIjoiY29uZmlndXJhdGlvbiIsInZlcnNpb24iOiJhZGQwYjRkYi01NGY4LTExZWEtODIwMi00YjA4MWU3NDBmNDQifQ.K5QHijc0oT71-eU5l4wfSnlUXAo4UrhLdVpA10eOvZ7yqVHt46wMtfOPZB5RQw33PhtQMcoK-bYkyZD3xrPv2RverRm502WR2xwZpmq7-aVxA4oNy1qD7o4WCfsNYjGekGgeS-xutW4yb20L3xCu5Lf-PnEGoEcsKvi_Vz0vUfnUrt2VQkcW-Wg175DehJgzYVvHpQcaOzBQqrI2Civ2cCVKy7RqIDuM-27Kqrd5WTD8qthHAo7op0Z0lID2g6uUul5usVCFyiVwUQW1gOr74uqb2j8ExxOjRnf55giJ9WJNJ9JWg6JPmkS5VJcMK3AhKRSBofyDTIlwiWeoGga1pw)"
+time="2020-02-25T17:42:15Z" level=info msg="Fetching from cache..." res=/opt/mule/conf/tokenpolicyapigw-4-2-2.yml
+time="2020-02-25T17:42:15Z" level=info msg="Fetching from cache..." res=/opt/mule/apps/tokenpolicyapigw-4-2-2.jar
+time="2020-02-25T17:42:25Z" level=warning msg="/opt/mule/conf/tokenpolicyapigw-4-2-2.yml: dial tcp: i/o timeout, retrying in 1s"
+time="2020-02-25T17:42:25Z" level=warning msg="/opt/mule/apps/tokenpolicyapigw-4-2-2.jar: dial tcp: i/o timeout, retrying in 1s"
+time="2020-02-25T17:42:36Z" level=warning msg="/opt/mule/apps/tokenpolicyapigw-4-2-2.jar: dial tcp 10.100.167.56:80: i/o timeout, retrying in 1.664560053s"
+time="2020-02-25T17:42:36Z" level=warning msg="/opt/mule/conf/tokenpolicyapigw-4-2-2.yml: dial tcp 10.100.167.56:80: i/o timeout, retrying in 1.437714187s"
+time="2020-02-25T17:42:48Z" level=warning msg="/opt/mule/conf/tokenpolicyapigw-4-2-2.yml: dial tcp: i/o timeout, retrying in 2.273912491s"
+time="2020-02-25T17:42:48Z" level=warning msg="/opt/mule/apps/tokenpolicyapigw-4-2-2.jar: dial tcp: i/o timeout, retrying in 3.060469218s"
+time="2020-02-25T17:43:00Z" level=warning msg="/opt/mule/conf/tokenpolicyapigw-4-2-2.yml: dial tcp 10.100.167.56:80: i/o timeout, retrying in 1.459459134s"
+time="2020-02-25T17:43:01Z" level=warning msg="/opt/mule/apps/tokenpolicyapigw-4-2-2.jar: dial tcp 10.100.167.56:80: i/o timeout, retrying in 2.095634783s"
+```
```

Comment:
@paigehargrave Can we remove this? This doesn't provide any context on the problem and is very misleading.

Reply:
Yes - I've removed this.